src/lib/browse/schemes/SchemesLayer.svelte

```diff
@@ -35,45 +32,37 @@
 
   let errorMessage = "";
 
-  let mainAtfName = "main_atf_schemes";
-  let mainAtfTitle = "ATF schemes";
-  let mainAtfShow = showHideLayer(mainAtfName);
-  let mainAtfStyle = "fundingProgramme";
-  $: [mainAtfColor, mainAtfLegend] = pickStyle(mainAtfStyle);
+  let atfName = "atf_schemes";
+  let atfTitle = "ATF schemes";
+  let atfShow = showHideLayer(atfName);
+  let atfStyle = "fundingProgramme";
+  $: [atfColor, atfLegend] = pickStyle(atfStyle);
 
-  let mainLcwipName = "main_lcwip_schemes";
-  let mainLcwipTitle = "LCWIP schemes";
-  let mainLcwipShow = showHideLayer(mainLcwipName);
-  let mainLcwipStyle = "interventionType";
-  $: [mainLcwipColor, mainLcwipLegend] = pickStyle(mainLcwipStyle);
+  let lcwipName = "lcwip_schemes";
+  let lcwipTitle = "LCWIP schemes";
+  let lcwipShow = showHideLayer(lcwipName);
+  let lcwipStyle = "interventionType";
+  $: [lcwipColor, lcwipLegend] = pickStyle(lcwipStyle);
 
-  let localAtfName = "local_atf_schemes";
-  let localAtfTitle = "Your ATF schemes";
-  let localAtfShow = showHideLayer(localAtfName);
-  let localAtfStyle = "fundingProgramme";
-  $: [localAtfColor, localAtfLegend] = pickStyle(localAtfStyle);
-
-  let localLcwipName = "local_lcwip_schemes";
-  let localLcwipTitle = "Your LCWIP schemes";
-  let localLcwipShow = showHideLayer(localLcwipName);
-  let localLcwipStyle = "interventionType";
-  $: [localLcwipColor, localLcwipLegend] = pickStyle(localLcwipStyle);
+  let localName = "local_schemes";
+  let localTitle = "Your schemes";
+  let localShow = showHideLayer(localName);
+  $: [localColor, localLegend] = pickStyle("interventionType");
```

Comment:
Wondering about local legend: v2 will want to be rebased onto this and we may want to rethink the legend

Reply:
Yep, let's figure that out in #544. There's more changes to browse needed for v2, like updating the intervention popup